.github/workflows/pull-request.yml

```
@@ -449,3 +449,139 @@
+          git config user.name "GitHub Actions Bot"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+      - name: Check and Commit Documentation
+        run: |
+          # Check if there are any changes
+          if [[ -n "$(git status --porcelain)" ]]; then
+            # Add all changes
+            git add docs/docs/auto-docs
+            git add docs/docs/user-guide
+            git add docs/docs/developer-guide
+            
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)